src/lib/common/limits.h

```diff
@@ -65,6 +65,8 @@
 #define IP_LENGTH_MAX           15    // Based on xxx.xxx.xxx.xxx
 #define STRING_SIZE_FOR_INT     16    // Room enough for an integer
 #define STRING_SIZE_FOR_DOUBLE  64    // Room enough for a double
+#define CORRELATOR_ID_SIZE      36
```

Comment:
CORRELATOR_ID_SIZE explains pretty well what it is all about. However, I'm still thinkg that it is valuable to explain from where that number comes (in the same sense that for example it is important to explain in the corresponding place that 24 size limits in subsId comes from MongoDB OID length).